AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -119,6 +119,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+mediamarkt.*###mms-consent-portal-container
```

Comment:
Cannot find `#mms-consent-portal-container`

Reply:
@Alex-302 You can find it in here: `www.mediamarkt.at` and also you can find `##.gdpr-cookie-layer--show` rule in `www.mediamarkt.nl`

Reply:
So `*` is not required here, if used on one domain only.

Reply:
`###mms-consent-portal-container` rule used in `mediamarkt.at` and `mediamarkt.de` sites.